Swift/KVSiOSApp/Constants.swift

```diff
@@ -1,12 +1,14 @@
 import Foundation
 import AWSCognitoIdentityProvider
 
-let cognitoIdentityUserPoolRegion = AWSRegionType.USWest2 //  <- REPLACE ME!
+// Cognito constants
+let awsCognitoUserPoolsSignInProviderKey = "UserPool"
+
+let cognitoIdentityUserPoolRegion = AWSRegionType.Unknown //  <- REPLACE ME!
```

Comment:
or we could do let cognitoIdentityUserPoolRegion = " REPLACE ME" //e.g. AWSRegionType. AWSRegionUSEast1 Ref: https://docs.aws.amazon.com/ja_jp/AWSiOSSDK/latest/Constants/AWSRegionType.html